web/src/app/modules/views/main/editors/modules/contents-container/services/content-container.service.ts

```diff
@@ -0,0 +1,18 @@
+import { EventEmitter, Injectable } from '@angular/core';
+import { IContainer } from '../../../../../../../model/IContainer';
+import { Subject } from 'rxjs';
+
+ @Injectable()
+export class ContentsContainerService {
+
+   private onModelDeleted = new Subject<string>();

```

Comment:
I'd prefer an EventEmitter for this. Currently, we refrain from using rxjs-components directly.